lib/rubocop/cop/infinum/factory_association.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'rubocop'
+
+module RuboCop
+  module Cop
+    module Infinum
+      # This cop looks for 'association' in factories that specify a ':factory' option
```

Comment:
Add the goal of this cop. Is it convention, performance optimization or preventing bugs?

Reply:
Dunno which goal to add, I mean it is performance optimisation, but not sure how to make it sound clear what it does but also that it is performance optimisation

Reply:
I'd look into the Slack thread and find something there 🙂 A sentence or two should suffice.